apps/learner/src/routes/content/[id]/quiz/+page.svelte

```diff
@@ -61,19 +97,68 @@
         >
           <span
             class={[
-              'flex items-center justify-center rounded-lg bg-slate-100 px-2 py-1 text-xs font-semibold text-black',
+              'rounded-lg bg-slate-100 px-2 py-1 text-xs font-semibold text-black',
               selectedOptionIndex === index && '!bg-slate-900 !text-white',
             ]}
           >
             {getOptionLetter(index)}
           </span>
-          <span class="text-left text-sm">
+          <span class="text-left text-base">
             {option}
           </span>
         </button>
       {/each}
     </div>
   </div>
 
-  <Button onclick={nextQuestion} disabled={selectedOptionIndex === -1}>Check Answer</Button>
+  <Button
+    onclick={() => {
+      checkAnswer();
+      showFeedbackModal();
+    }}
+    disabled={selectedOptionIndex === -1 || isFeedbackModalOpen}>Check Answer</Button
+  >
+
+  <div
```

Comment:
Can consider use the boolean to conditionally render the modal instead of rendering the modal and uses CSS to hide it.